website/src/pages/index.js

```diff
@@ -127,8 +136,29 @@ function Home() {
             </div>
           </section>
         )}
+        {showcase && showcase.apps.length > 0 && (
+          <section className='container'>
+            <div className={clsx('row', styles.showcaseTitleContainer)} >
+              <div className={styles.divider} style={{ marginRight: 10 }} />
+              <h1 className={styles.showcaseTitle}>{`See what you can build\nwith Wave`}</h1>
```

Comment:
There should be just a single `h1` per page in general. Please adjust the rest of the headings accordingly.
```suggestion
              <h2 className={styles.showcaseTitle}>What can Wave achieve?</h2>
```